demisto_sdk/commands/pre_commit/hooks/docker.py

```diff
@@ -319,6 +367,7 @@ def get_new_hooks(
             hook.pop("config_file_arg", None)
             hook.pop("copy_files", None)
             hook.pop("split_by_object", None)
+            hook.pop("split_by_network", None)
```

Comment:
split_by is not a good name since its an implementation detail, not a feature
we should find a name that describes the fact that we will listen to network settings from the pack ignore

Reply:
I agree. Do you have a suggestion? Both for `split_by_object` and `split_by_network`
How about `run_each_object` and `respect_networking_config`?